gossipd/gossip.c

```diff
@@ -1641,8 +1669,11 @@ static void try_connect(struct reaching *reach)
 	}
 
 	a = find_addrhint(reach->daemon, &reach->id);
+
+	if (!a)
+		a = seed_resolve_addr(reach, &reach->id, 9735);
+
 	if (!a) {
-		/* FIXME: now try node table, dns lookups... */
```

Comment:
We now do the BOLT10 DNS lookup, but maybe node table is still not used? I think, "node table" here is the gossip-derived map of nodes, which contain `struct wireaddr` to contact them with?